### DISCUSSION TITLE
How to delete old worlds and avatars preview pictures from vrcx

### DISCUSSION THREAD

**Discussion by LilacGiant** [vrcx-team/VRCX#612] [Q&A] [answered]
Is there a way to delete old uploaded worlds and avatars preview pictures from vrcx? Where are they stored?

**Comment by Myrkie** [comment]
as far as im aware VRCX doesn't have this functionally, only the ability to add.

**Comment by Nekromateion** [comment, marked as answer]
The pictures aren't stored locally.
VRChat keeps the history of the files.
Example:
LS Media is currently on their 19th image URL ( https://api.vrchat.cloud/api/1/file/file_7b9a68b0-13ea-4d27-bdfc-71a20ee49a41/19/file )
You can simply change the number to get a previous version ( https://api.vrchat.cloud/api/1/file/file_7b9a68b0-13ea-4d27-bdfc-71a20ee49a41/6/file ) to get one of the older versions.

As far as I know VRChat does not allow you to delete these.
